ontologies/WIP/sierratransformable

```diff
@@ -100,84 +100,136 @@
-   * 5. For "110" type as "Organisation"
-   * 6. For "111" type as "Event"
-   * 7. If subfield 0 contains a value to use to populate "identifiers" after stripping first two letters (ls, sh). Check we want to do this. Platform "authority" is defined by first two letters of code; n or nr. Indicators can also help disabiguate the authority. 
-   * 8. For all other subfields populate "qualifiers" apart from 6,9,4 populate "qualifierType" with the subfield type (kebab-case) 
-   * 9. For subfield "d" and "f" type as "Period", for all other subfields provide no type
-   * 10. If 700, 710 or 711 repeat with same rules where; 700 type Person, 710 type Organisation, 711 type Event 
-   * 11. Inaddition to excluding 6,9,4 also exclude 4 and 5 
-   *
-   * Note: need to remove trailing punctuation on subfields. The punctuation is there for the purpose of the concatentation of concept and qualifiers.
-   *
-   *
-   * Example:
+   * 1. For all bibliographic records use "marcTag": "100", "110" (removing this from scope "111")
+   * 2. Platform "name" populate from subfield "a"
+   * 3. For "100" type as "Person" populate "prefix" from 100 subfield c and "numeration" from b 
+   * 4. For "110" type as "Organisation"
+   * 5. Ignore: For "111" type as "Event"
+   * 6. If subfield 0 contains a value use it to populate "identifiers". For people the "identifierScheme" will be lc-names. Note: I am seesing variable spaces in id's we most likely will need to remove these.
+   * ignore for now 7. For subfield "d" and "f" type as "Period", for all other subfields provide no type
+   * ignore for now 9. If 700, 710 or 711 repeat with same rules where; 700 type Person, 710 type Organisation, 711 type Event 
+   * ignore for now 10. Inaddition to excluding 6,9,4 also exclude 4 and 5 
+   *
+   * Note: Do we need to remove trailing punctuation on subfields? The punctuation is there for the purpose of the concatentation of concept and qualifiers.
+   *
+   * Example person:
    *  "creators": [
    *     {
-   *        "label": "Jacobs, Philip P. (Philip Peter), 1879-1940.",
+   *        "id": "b26h7qf3"
+   *        "name": "Jacobs, Philip P.",
+   *        "prefixes": ["Sir"] ,
+   *        "numeration": "II" ,
    *        "type": "Person" , 
    *        "identifiers": [
    *        {
    *           "value": "2005016805",
    *           "type": "Identifier",
-   *           "identifierScheme": "lcsh"
+   *           "identifierScheme": "lc-names"
    *        }
    *        ]
    *     } 
    *     ]
    *
+   * Example organisation:
+   *  "creators": [
+   *     {
+   *        "id": "8t677qf3"
+   *        "name": "International Labour Organisation",
+   *        "type": "Organisation" ,
+   *        "identifiers": [
+   *        {
+   *           "value": "2005016805",
+   *           "type": "Identifier",
+   *           "identifierScheme": "lc-names"
+   *        }
+   *        ]
+   *     } 
+   *     ]
+   *
+   
+   /* Populate wwork:contribution. Rules:
```

Comment:
Is it contribution or contributors? Example below users "contributors".

Reply:
Changing top to accurately state contributors